generator/tree.go

```diff
@@ -107,6 +107,16 @@ func metricType(t string) (string, bool) {
 	}
 }
 
+func metricAccess(a string) (string, bool) {
+	switch a {
+	case "ACCESS_READONLY", "ACCESS_READWRITE":
```

Comment:
Yes, according to the [RFC 2578](https://tools.ietf.org/html/rfc2578#section-7.3), `read-create` is the superset of `red-write`, many thanks.